jenkinsfiles/RHEL_Check.groovy

```diff
@@ -285,29 +283,40 @@ pipeline {
                             script {
                                 try {
                                     withEnv(readFile('envs_tobe_loaded.txt').split('\n') as List) {
-                                        def gamma_return_value = sh(
-                                                script: """
-                                                        cd oneAPI-samples/Libraries/oneDPL/gamma-correction/
-                                                        mkdir build
-                                                        cd build/
-                                                        cmake ..
-                                                        make
-                                                        make run
-                                                        exit \$?""",
-                                                returnStatus: true, label: "gamma_return_value Step")
-                                        def stable_sort_return_value = sh(
-                                                script: """
-                                                        cd oneAPI-samples/Libraries/oneDPL/stable_sort_by_key/
-                                                        mkdir build
-                                                        cd build/
-                                                        cmake ..
-                                                        make
-                                                        make run
-                                                        exit \$?""",
-                                                returnStatus: true, label: "stable_sort_return_value Step")
-
-                                        if (gamma_return_value != 0 || stable_sort_return_value !=0) {
-                                            echo "gamma-correction or stable_sort_by_key check failed. Please check log to fix the issue."
+                                        def gamma_return_value = runExample("gamma_correction")
+                                        def stable_sort_return_value = runExample("stable_sort_by_key")
+                                        def convex_hull_return_value = runExample("convex_hull")
+                                        def dot_product_return_value = runExample("dot_product")
+                                        def histogram_return_value = runExample("histogram")
+                                        def random_return_value = runExample("random")
+
+                                        def test_pass_status = true
+                                        if (gamma_return_value != 0) {
+                                            echo "gamma-correction check failed."
+                                            test_pass_status = false
+                                        }
```

Comment:
Is it possible to "pack" these parts into `runExample` function as well? Similar to Windows.

Reply:
Done